### PR TITLE
Admissions student profile views block update

### PR DESCRIPTION
**Pull request by pyrello**
Pulled out from #6054 

**Note:** When this is rolled out, it will cause a slight visual regression and will require updating  https://admissions.uiowa.edu/international-student-ambassadors, https://admissions.uiowa.ddev.site/california-hawkeyes and https://admissions.uiowa.ddev.site/texas-hawkeyes as soon as the deployment is done.

# How to test
```
ddev blt ds --site admissions.uiowa.edu && ddev drush @admissions.local uli node/7331/layout
```

- Remove existing "Student Profiles Image Grid" block
- Add "Student Profiles Card Grid" block, setting the list format to Grid, and the grid layout to 3 column
- Save the layout
- Compare with https://admissions.uiowa.edu/international-student-ambassadors
- Ensure no changes on https://admissions.uiowa.ddev.site/student-profiles vs https://admissions.uiowa.ddev.site/student-profiles
- Confirm that image grid on student profile pages are still working https://admissions.uiowa.ddev.site/student-profiles/ainsley-hiestand

## Page titles

- Test that these page titles have gold highlighted background.   The student profile should be using Zilla, and the other two are using Antonio.

https://admissions.uiowa.ddev.site/academics/2plus2/anthropology-northeast-iowa-community-college
https://admissions.uiowa.ddev.site/academics/accounting
https://admissions.uiowa.ddev.site/student-profiles/sophia-restiffe-favoretto

### CODE REVIEW

**Comment by bspeare** [comment]
The profiles blocks look good.  I added a few items that were missed in https://github.com/uiowa/uiowa/pull/6203/files.

File: config/sites/admissions.uiowa.edu/config_split.patch.layout_builder_styles.style.list_format_list.yml
Comment (bspeare) [inline config/sites/admissions.uiowa.edu/config_split.patch.layout_builder_styles.style.list_format_list.yml:1]: I had remove the list option from the blocks in https://github.com/uiowa/uiowa/blob/d6308b372d28cdc73f83d796eefa3742deb79b2b/config/sites/admissions.uiowa.edu/config_split.patch.layout_builder_styles.style.list_format_list.yml, do you want to keep that option here?

File: docroot/sites/admissions.uiowa.edu/modules/admissions_core/admissions_core.module [hunk 1]
```diff
@@ -407,6 +407,33 @@ function admissions_core_preprocess_field(&$variables) {
...
+  if ($variables['element']['#bundle'] === 'area_of_study') {
+    // Remove existing page title classes.
+    $variables['attributes']['class'] = [];
+    // Add new page title classes.
+    uiowa_core_set_attributes($variables, [
+      'headline--uppercase',
+      'headline--negative',
+      'headline--highlight',
+    ]);
+  }
```
Comment (pyrello) [inline docroot/sites/admissions.uiowa.edu/modules/admissions_core/admissions_core.module:427]: This seems unrelated to this update.

Comment (bspeare) [reply docroot/sites/admissions.uiowa.edu/modules/admissions_core/admissions_core.module:427]: This fixes work we missed in https://github.com/uiowa/uiowa/pull/6203/files.

**Review by bspeare — APPROVED** [review]
Tested locally and it works!
